configuration/pih/htmlforms/hiv/section-hiv-plan.xml

```diff
@@ -1,6 +1,6 @@
 <htmlform id="hiv-plan"
-          formUuid="e86e3118-731c-11e7-8cf7-a6006ad3dba0" formName="HIV Plan"
```

Comment:
One annoyance is that message properties for these form(lets) use this uuid.  So when you change the uuid for different versions, you also need to add a new message to message properties (and translate).  It will use the formName, but unless you add to message properties, it won't provide the translations.  As I am reviewing this file, I will add the uuids to the message property file.

ui.i18n.Form.name.e86e3118-731c-11e7-8cf7-a6006ad3dba0=HIV Plan

Reply:
And sorry for the long delay...